mathics/builtin/__init__.py

```diff
@@ -121,3 +147,9 @@ def contribute(definitions):
         if not definitions.have_definition(ensure_context(operator)):
             op = ensure_context(operator)
             definitions.builtin[op] = Definition(name=op)
+
+    
+#    definitions.builtin['System`$PrePrint'].set_values_list('messages',
+#                                                            Rule('usage',"$PrePrint is a global variable whose value, if set, is applied to \
+#every expression before it is printed. ",True))
+    
```

Comment:
I left this because it was another possibility to check if the hook was defined. I removed the comment now.